upi/azurestack/05_masters.json

```diff
@@ -0,0 +1,254 @@
+{
+  "$schema" : "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion" : "1.0.0.0",
+  "parameters" : {
+    "baseName" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Base name to be used in resource names (usually the cluster's Infra ID)"
+      }
+    },
+    "masterIgnition" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "Ignition content for the master nodes"
+      }
+    },
+    "numberOfMasters" : {
+      "type" : "int",
+      "defaultValue" : 3,
+      "minValue" : 1,
```

Comment:
Is this something that's only for ASH, or should we bring the min number of Masters to 1 in Azure as well?

Reply:
this probably shouldn't be a parameter at all (it is always 3)